nutils/evaluable.py

```diff
@@ -4437,6 +4437,10 @@ def _node(self, cache, subgraph, times, unique_loop_ids):
         cache[self] = node = self._node_loop_body(loopcache, loopgraph, looptimes, unique_loop_ids)
         return node
 
+    @cached_property
+    def arguments(self):
+        return super().arguments - frozenset({self.index})
+
```

Comment:
The commit message explains that the `Loop.arguments` serves to shadow the default implementation, yet it calls `super().arguments`. What am I missing?

Reply:
Fixed.